.env.local

```
@@ -8,8 +8,22 @@ TWILIO_AUTH_TOKEN=ACb805d62236fe8d625c4c4897bd96a6a5
+BIZCIVITAS_BACKEND_URL=https://dev-backend.bizcivitas.com/api/v1
+NEXT_PUBLIC_BACKEND_URL=https://backend.bizcivitas.com/api/v1
 
 WHATSAPP_API_KEY=9ddf7dca27344afd80543dccd5e9b096
 WHATSAPP_EVENT_TEMPLATE_NAME=event_register
-WHATSAPP_PAYMENT_TEMPLATE_NAME=membership_success
\ No newline at end of file
+WHATSAPP_PAYMENT_TEMPLATE_NAME=membership_success
+
+# MongoDB Configuration (add your MongoDB connection string)
+MONGODB_URI=mongodb+srv://connect:TFo579aPtj8xRU0B@bizcivitas-register.14gin.mongodb.net/?retryWrites=true&w=majority&appName=Bizcivitas-register
+MONGODB_DB_NAME=test
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MONGODB_DB_NAME key should go before the MONGODB_URI key

(UnorderedKey)